apps/web/app/environments/[environmentId]/surveys/[surveyId]/edit/ResponseOptionsCard.tsx

```diff
@@ -36,12 +41,52 @@ export default function ResponseOptionsCard({ localSurvey, setLocalSurvey }: Res
     setLocalSurvey({ ...localSurvey, redirectUrl: link });
   };
 
+  const handleCloseSurveyMessageToggle = () => {
+    if (surveyClosedMessageToggle && localSurvey.surveyClosedMessage) {
```

Comment:
What do you think about this refactoring suggestion by chatGPT

`  setSurveyClosedMessageToggle((prev) => !prev);
  if (surveyClosedMessageToggle && localSurvey.surveyClosedMessage) {
    setLocalSurvey((prev) => ({ ...prev, surveyClosedMessage: null }));
  }
};`